internal/app/connectconformance/testsuites/basic.yaml

```diff
@@ -333,6 +436,149 @@ testCases:
         - name: x-custom-trailer
           value: ["bing","quux"]
       fullDuplex: true
+- request:
```

Comment:
I think it would be nice to keep the timeouts and cancellations in separate files and have "basic" just be basic happy-path RPCs.

Also, I think for the cancellations file, we'd want `mode: TEST_MODE_CLIENT` at the top, since there's not much use in running these test cases against a server under test, since we can't actually assert anything about how the server behaves in the face of cancellation -- all we can really see/verify is that the client-side RPC framework fails the call with a "cancelled" code.